x-pack/plugins/case/server/routes/api/cases/configure/get_configure.test.ts

```diff
@@ -122,4 +125,41 @@ describe('GET configuration', () => {
     expect(res.status).toEqual(404);
     expect(res.payload.isBoom).toEqual(true);
   });
+
+  it('returns an error when mappings request throws', async () => {
+    const req = httpServerMock.createKibanaRequest({
+      path: CASE_CONFIGURE_URL,
+      method: 'get',
+    });
+
+    const context = await createRouteContext(
+      createMockSavedObjectsRepository({
+        caseConfigureSavedObject: mockCaseConfigure,
+        caseMappingsSavedObject: [],
+      })
+    );
+    const mockGetMappings = jest.fn().mockImplementation(() => {
+      throw new Error();
+    });
```

Comment:
nit -> As you don't `expect` the function to be called it can be written as:
```suggestion
    const mockGetMappings = () => {
      throw new Error();
    };
```